app/models/post.rb

```diff
@@ -0,0 +1,18 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :comments, dependent: :destroy
+  has_many :likes, dependent: :destroy
+
+  after_create :post_counter
```

Comment:
- [ ] It would be better to use the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - after_save to call the #update_post_counter method every time a new post has been saved to the database 👍🏿